packages/cli/axe-cli

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require('./index')
\ No newline at end of file
+require('./dist/src/bin/cli.js')
```

Comment:
I've never understood the purpose of this file. Why not just point `package.json#bin` to the `cli.js` file?